examples/md-flexible/scripts/plotTuning.py

```diff
@@ -14,13 +14,22 @@
 # help message
 for arg in sys.argv[1:]:
     if "--help" in arg:
-        print("Usage: ./plotTuning.py [path/To/mdFlex/std.out ...]")
+        print("Usage: ./plotTuning.py [path/To/mdFlex/std.out ...] or [path/To/mdFlex/directoryWithOutput]")
         print("If no input is given the script looks for the latest testTuning directory in the current directory.")
         exit(0)
 
 # take all input files as source for a plot
 if len(sys.argv) > 1:
-    datafiles=sys.argv[1:]
+    if ".out" in arg:
```

Comment:
better check if the argument is a directory or file, otherwise, this will only work properly if one always goes for the `.out` filename, which is completely arbitrary.